glut/Makefile.am

```diff
@@ -67,8 +67,7 @@ endif
 
 test_glut_SOURCES=test_glut.c
 test_glut_LDADD=$(addprefix $(top_builddir)/, $(LIBS_MGH)) $(LIBS_VXL) -lz \
-	$(LIB_JPEG) $(LIB_TIFF)	$(X_LIBS) $(LIBS_OPENGL) $(top_builddir)/glut/libglut.a
-
+	$(LIB_JPEG) $(LIB_TIFF)	$(top_builddir)/glut/libglut.a $(X_LIBS) $(LIBS_OPENGL) 
```

Comment:
I get two errors:

1) The job exceeded the maximum time limit for jobs, and has been terminated.

2) No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

I think I can fix (2). And I might be able to fix (1). But I'm confused, how does it know to split into 2 groups of tests. I see you added matrix, with 2 entries, but those list are incomplete. The tests it fails on isnt even included in that list - so Im not sure its even recognizing it.